bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/FlatRow.java

```diff
@@ -128,13 +128,22 @@ public boolean equals(Object obj) {
         return false;
       }
       Cell other = (Cell) obj;
-      return Objects.equal(family, other.family) &&
-          Objects.equal(qualifier, other.qualifier) &&
-          timestamp == other.timestamp &&
+      return hasEqualKeys(other) &&
```

Comment:
I updated hasEqualKeys to equalFamilyQualifierAndTimestamp.

I updated the comment.

The logic new method was an IDE refactoring of the three tests and added `other != null` which will not change the functionality of the original code in `equals`.